packages/fiber/src/native/Canvas.tsx

```diff
@@ -145,9 +210,14 @@ const CanvasImpl = /*#__PURE__*/ React.forwardRef<View, Props>(
     }, [canvas])
 
     return (
-      <View {...props} ref={viewRef} onLayout={onLayout} style={{ flex: 1, ...style }} {...bind}>
+      <View {...props} ref={viewRef} onLayout={onLayout} style={{ flex: 1, ...style }}>
```

Comment:
This should allow props to be spread from libraries like https://github.com/TiagoCavalcante/r3f-native-orbitcontrols which also use responder props.